src/server/services/csvPipeline/success.js

```diff
@@ -2,6 +2,8 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+const express = require('express')
```

Comment:
I think (guess) it is okay to add a require for the comment. However, if we are going to do that then I would also do it in failure.js as it seems to be the same usage there.

One final thought is to put a comment on the require to indicate it is needed to resolve the type in the comment so it is clear for future developers.